### ISSUE TITLE
UI bug when query returns nothing for creating new Alert (Unified Alerting 9.2.3)

### ISSUE THREAD

**Issue by AlexeiZenin** [grafana/clickhouse-datasource#281]
**Describe the bug**
Get an error page in the Grafana UI when trying to create an alert on a time series which returns no results from clickhouse.

**Versions**
Grafana 9.3.2 and Plugin version 2.0.3

**Query**
Errors in the last X minutes
```sql
SELECT toStartOfInterval(timestamp, INTERVAL 20 second) as time, serviceName, count(*) as numErrors
FROM x.x
WHERE serviceName = 'x' AND $__timeFilter(timestamp) AND hasError = true
AND responseStatusCode != '' AND toInt16(responseStatusCode) >= 500
GROUP BY time, serviceName
ORDER BY time
```

**To Reproduce**
Create a time series which returns no data in the alert panel, and click Run Query a few times until you get the error page. See screenshots

**Expected behavior**
Should not error out as time series should not always be expected to return a value and return some default set of values we can use with Alerting. When the time series has data (their are errors within whatever time period, it doesn't error out)

**Screenshots**
After clicking on Run Query the 1st time:
![Screen Shot 2023-01-18 at 12 01 43 PM](https://user-images.githubusercontent.com/12644364/213246639-edc4523c-1a9a-441d-9966-6aa55ce0cc99.png)
After clicking on Run Query the 2nd time:
![Screen Shot 2023-01-18 at 12 00 37 PM](https://user-images.githubusercontent.com/12644364/213246417-6f8e8777-68d0-46e1-a1b9-68a41e7afeb9.png)
![Screen Shot 2023-01-18 at 12 00 49 PM](https://user-images.githubusercontent.com/12644364/213246420-faa8f830-ccf1-4fb9-bf63-5fbbc6571e9d.png)

**Comment by bossinc** [comment]
Hello, I believe you can only return numeric values. Try removing  `serviceName` from your example and see if you get the same issue. 🙂

**Comment by AlexeiZenin** [comment]
> @AlexeiZenin did removing `serviceName` resolve this or is there still an issue?

I didn't try this but for my requirements I need to keep serviceName to have the query use that as the label value for the alert so I can easily query for alerts for certain services. 

According to the docs it should still work correct? https://grafana.com/grafana/plugins/grafana-clickhouse-datasource/
```
To create multi-line time series, the query must return at least 3 fields in the following order:

field 1: datetime field with an alias of time
field 2: value to group by
field 3+: the metric values
```

**Comment by AlexeiZenin** [comment]
My workaround was to due some ugly unions on a default row when no errors were there where I would create a time series with 1 row with value 0

**Comment by bossinc** [comment]
This seems related to #207 I am looking into this more. Thank you for the update @AlexeiZenin

**Comment by bossinc** [comment]
We are still aware of this issue. I'll have time to look into this next week. I'll have an update then :slightly_smiling_face:

**Comment by bossinc** [comment]
@AlexeiZenin this issue is fixed as of 3.0.1-beta 😄 
Please reopen if you have more questions.
